hijack/tests/test_hijack.py

```diff
@@ -362,3 +362,8 @@ def test_middleware_with_remote_user(self):
         request.user = self.superuser
         middleware.process_request(request)
         self.assertEqual(request.META.get('REMOTE_USER'), self.superuser_username)
+
+    def test_missing_session(self):
+        with SettingsOverride(settings, MIDDLEWARE=()):
+            response = self.client.get('/hello/')
```

Comment:
@khpeterson You could also test these changes more directly by calling the `is_hijacked` and `hijack_notification` functions directly in your test. Use the django request factory to make the request objects to pass to them.

Reply:
OK, good idea, done.